### DISCUSSION TITLE
What is the meaning of "Returning whistleblowers" and "Anonimity" diagrams?

### DISCUSSION THREAD

**Discussion by taktgeber** [globaleaks/globaleaks-whistleblowing-software#4285] [Q&A] [answered]
Hi everyone,

could someone explain to me in detail what is behind the following statistics:
![image](https://github.com/user-attachments/assets/f5fca838-3ad3-449d-be21-c43f995b235d)

Thank you very much for your help!

best regards

**Comment by evilaliv3** [comment, marked as answer]
Hello @taktgeber 

Thank fou for your question.
The first diagram shows how many are the whistleblowers that logged in via their receipt versus the ones that never logged in after filing the report.
The second details how many are the users that files the report without entering their details (so anonymousle), how many entered their details in their initial report, and how many entered their details in a subsequent moment. This second diagram is related to the questions "Would you like to tell us who you are"

**Reply by taktgeber** [reply]
Hi @evilaliv3,
Thank you very much!
Is there a possibility to customize these diagrams or create own ones?
And am I able to ask the question "Would you like to tell us who you are?" in an own questionnaire. And how to configure the question so that it triggers this function?
Best regards
